systemd_ctypes/introspection.py

```diff
@@ -19,7 +19,7 @@
 
 def parse_method(method):
     return {
-        "in": [tag.attrib['type'] for tag in method.findall("arg[@direction!='out']")],
+        "in": [tag.attrib['type'] for tag in method.findall("arg[@direction='in']")],  # != is Python 3.10 only
```

Comment:
This change is a no-go.  The spec states that if the direction is left out, it needs to be interpreted as `in`.

Reply:
Ah, good point, bit inelegant fix but seems to work.